goagen/gen_client/generator.go

```diff
@@ -615,6 +617,84 @@ func (g *Generator) generateUserTypes(pkgDir string) error {
 	return utWr.FormatCode()
 }
 
+// mediaTypeMetadataImports will grab all imports specified in a struct:field:type Metadata tag and
+// add them to the imports for the media_types.go file.
+func (g *Generator) mediaTypeMetadataImports(s []*codegen.ImportSpec) []*codegen.ImportSpec {
```

Comment:
Please move the `attributeImports` function described above to the `codegen` package and use it from both the `genapp` and `genclient` package instead of copy/pasting.